samples/basic/basic.js

```diff
@@ -145,3 +146,24 @@ function performTransformations() {
   console.log("> That's it. You can now open the URLs above in a browser");
   console.log("> and check out the generated images.");
 }
+
+//using async/await with cloudinary upload
```

Comment:
@rebeccapeltz , you added that method, but what's calling it?

Reply:
see in this function  the `await open(url) . `  ```async function uploadImage(filename) {
  try {
      const result = await cloudinary.uploader.upload(filename, { tags: 'basic_sample' });
      console.log(result)
      const url = await cloudinary.url(result.public_id, {
          width: 400,
          quality: 'auto',
          fetch_format: 'auto',
          secure: true
      });
      console.log(url)
      await open(url)
  }
  catch (err) {
      console.log(err)
  }
}```

Reply:
Open is a function from https://www.npmjs.com/package/open . I see you are providing instructions to open in browser.  The open function opens in default browser.